functions/check-started/index.js

```diff
@@ -129,5 +129,5 @@ function canCheckBeStarted (code) {
 async function requestPupilStatusChange (pupilId, checkCode) {
   const pupilStatusQueueName = 'pupil-status'
   const message = { version: 1, pupilId, checkCode }
-  return await azureStorageHelper.addMessageToQueue(pupilStatusQueueName, message)
+  return azureStorageHelper.addMessageToQueue(pupilStatusQueueName, message)

```

Comment:
Why remove the await?

Reply:
It is an async function so any callers would already `await` on the Promise.  There is no need to await internally and for the caller to also await - it just takes extra time.  A single await is enough.